### ISSUE TITLE
[DDL][Board Appeals] List View

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/va.gov-team#55272]
## Value Statement
**_As a_** veteran waiting on my board of appeals decision
**_I want to_** be able to distinguish my appeals decision from other letters in the decision letter list view
**_So that_** I'm able to easily find and download that letter.

---
## Background Context
-Currently veterans have to wait for their Board of Appeals decision letters in snail mail. BVA is eager to surface these letters within the DDL feature. According to our contact at CaseFlow, we would only need to filter by doc ID "27" to expose these letters to end users. That doc type does not seem to be associated with any other documents other than Board of Appeals decision letters.

## Acceptance Criteria

- [x] Board of Appeals letters appear in the DDL List View
- [x]  Board of Appeals letters are appropriately distinguished from other Notification Letters are can be clearly identified as what they are
- [x]  Clicking on a download link downloads the correct document type, a Board of Appeals letter pdf
- [x] Meets accessibility standards
- [x] Is behind a feature toggle 

## Designs and Build Notes
- Add here

## Out of scope
- Add here

## Open questions and assumptions

- We can refer to the metadata to populate the list text
- Doc type ID only shows board of appeals letters
- Should we add a separate GA event for this?

## Tasks
- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## Definition of Ready 

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [x] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [x] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Reviewed and approved by product and/or design

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by jerekshoe** [comment]
@samcoforma this can be closed can't it?
